### ISSUE TITLE
On push, find audio in custom meta field

### ISSUE THREAD

**Issue by rellison959** [nprds/nprapi-wordpress#10]
To detect where audio URL is saved in META.

add one for text box to the push config, asking for the name of a custom meta that a site uses to store an audio href. On every push, it’ll grab that meta and add it to the Xml being sent to the API.

**Comment by eobanb** [comment]
For the life of me I cannot get this to work. I've installed this branch, set the custom field in the settings, created a post, added the audio (just a string of the mp3 URL), checked the 'send to NPR one' checkbox, and although the post gets ingested, the audio does not get attached.

**Comment by CrookedNumber** [comment]
Hmmm ... This might not be the right solution for you. I created this branch just for WMFE, but abandoned it when Ryan no longer needed something this specific.

How do you usually attach audio to posts? (the more details, the better)

**Comment by eobanb** [comment]
We use Podpress, which basically just creates a meta field for the post with the key 'enclosure' with the URL to the audio file and some other metadata. Here's an example:

http://wfiu.indiana.edu/podcasts/audio/amos/15/114.mp3
966526
audio/mpeg
a:1:{s:8:"duration";s:7:"0:02:00";}

**Comment by rellison959** [comment]
I was looking to possibly utilize this plugin as well. So a proper solution would be great!

Thanks,
Ryan Ellison
WMFE

**Comment by eobanb** [comment]
I actually thought WP-DS-NPR-API already supported Powerpress as of [the 1.5.2 release](http://digitalservices.npr.org/post/v152-api-wordpress-plug-now-available-0)...but I can't get it to attach the audio.

**Comment by CrookedNumber** [comment]
> http://wfiu.indiana.edu/player.php?myMediaURL=http://wfiu.indiana.edu/podcasts/audio/artsInterviews/15/150602-aparnanancherla-web.mp3

Indeed, it should . And I push OK with a local WP install + PowerPress. Closing this -- and dealing with it via DPRDS ticketing system. Will open a separate issue re: PowerPress if no luck.
